pkg/logs/logs.go

```diff
@@ -41,6 +43,8 @@ func Start() error {
 		return nil
 	}
 
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Why did you put that here ? should add it to the `init` method of the client package ?